assets/js/modules/analytics-4/datastore/settings.test.js

```diff
@@ -513,6 +533,36 @@ describe( 'modules/analytics-4 settings', () => {
 					status: 200,
 				} );
 
+				fetchMock.postOnce( fpmSettingsEndpoint, ( url, opts ) => {
+					const {
+						data: {
+							settings: { isEnabled },
+						},
+					} = JSON.parse( opts.body );
+
+					return {
+						body: {
+							isEnabled, // Return the `isEnabled` value passed to the API.
+							isFPMHealthy: true,
+							isScriptAccessEnabled: true,
+						},
+						status: 200,
+					};
+				} );
+
+				registry.dispatch( CORE_SITE ).setFirstPartyModeEnabled( true );
+				await registry.dispatch( MODULES_ANALYTICS_4 ).submitChanges();
+
+				expect( fetchMock ).toHaveFetched( fpmSettingsEndpoint, {
+					body: {
+						data: {
+							settings: { isEnabled: true },
+						},
+					},
+				} );
+			} );
+
+			it( 'should handle an error when sending a POST request to the FPM settings endpoint', async () => {
```

Comment:
As above, I added a couple of additional tests to extend the coverage for the existing behaviour.